packages/components/src/tooltip/README.md

```diff
@@ -35,12 +35,21 @@ The amount of time in milliseconds to wait before showing the tooltip.
 -   Required: No
 -   Default: `700`
 
+#### `placement`: `'top' | 'top-start' | 'top-end' | 'right' | 'right-start' | 'right-end' | 'bottom' | 'bottom-start' | 'bottom-end' | 'left' | 'left-start' | 'left-end'`
+
+Used to specify the tooltip's position with respect to its anchor.
```

Comment:
We could add a reference to the placement docs:

https://floating-ui.com/docs/tutorial#placements

Reply:
I grabbed this from Popover's README for consistency: https://github.com/WordPress/gutenberg/blob/ab97d28a55b028139b66b8199d795445d830cad8/packages/components/src/popover/README.md?plain=1#L199-L201

So maybe we'd want to add the link there, too? And after mentioning URLPopover above, I looked at the README on `placement`:

https://github.com/WordPress/gutenberg/blob/ab97d28a55b028139b66b8199d795445d830cad8/packages/block-editor/src/components/url-popover/README.md?plain=1#L97-L103

So I wonder if we should follow up with these and other components moving towards `placement`.

Reply:
Sounds like a good documentation improvement 👍 If these are coming from an external library, then IMHO it makes a lot of sense to link to the original library's documentation.

Reply:
The counter-argument to that, is that we're exposing our internal implementation in our docs. We usually avoid doing this as it gives us more flexibility in changing underlying implementation (like when `Popover` was refactored from `popper` to `floating-ui` last year)